src/components/flowchart/index.js

```diff
@@ -548,6 +548,7 @@ const emptyNodes = [];
 const emptyGraphSize = {};
 
 export const mapStateToProps = (state, ownProps) => ({
+  clickedNode: state.node.clicked,
   centralNode: getCentralNode(state),
```

Comment:
optional: might be worth adding a comment that the `centralNode` element will not be needed and replaced by  `clickedNode` once the lazy flag is removed?